models/eqV2_tracegrad/test_fairchem_kappa_debug.py

```
@@ -0,0 +1,365 @@
+                ) as file:
+                    json.dump(test_metrics, file, ensure_ascii=False, indent=4)
+
+
+def run_kappa(
+    checkpoint_path: Annotated[Path, typer.Option(help="路径，包含 checkpoint.pt")],
+    out_path: Annotated[Path, typer.Option(help="Output path to write results files")],
+    model_name: Annotated[str, typer.Option(help="输出目录的子目录名")],
+    identifier: Annotated[str, typer.Option(help="运行标识，用于文件夹命名")],
+    atom_disp: Annotated[float, typer.Option(help="用于计算声子性质的位移距离")] = 0.03,
+    seed: Annotated[int, typer.Option(help="随机种子")] = 42,
+    num_jobs: Annotated[int, typer.Option(help="并行任务数")] = 10,
```

Ruff (0.14.3)

Unused function argument: `num_jobs`

(ARG001)